index.html

```diff
@@ -1275,6 +1549,101 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
+It is best practice that JSON data structures typically do not expect changing types of
+their internal attributes. JSON-LD has implicit support for compact form serialization
+which transforms arrays with a single element only to switch its data type. Developers
+writing parsers have to implement special handling of these data types, which results in
+more code, is more error-prone and sometimes does not allow parsers based on code
+generation, which rely on static types.
+          </dd>
+          <dt id="pf25">
+PF25: Broad library support.
+          </dt>
+          <dd>
+JWT and JSON due to its maturity and standardization, have a lot of open-source library
+support. While JSON-LD 1.0 is a standard and has support for different programming
+languages, it is still behind JSON which is often part of the native platform
+toolchain, e.g., JavaScript. For LD-Proofs, on the other hand, only a few scattered
+libraries exist.
+          </dd>
+          <dt id="pf26">
+PF26: Easy to understand what is signed.
+          </dt>
+          <dd>
+JWT makes it visible what is signed in contrast to LD-Proofs, e.g., LD Signatures, that
+are detached from the actual payload and contain links to external documents which makes
+it not obvious for a developer to figure out what is part of the signature.
+          </dd>
+          <dt id="pf27">
+PF27: Ability to be used as authn/authz token with existing systems.
+          </dt>
+          <dd>
+Many existing applications rely on JWT for authentication and authorization purposes.
+In theory, developers maintaining these applications could leverage JWT-based verifiable
+presentations in their current systems with minor or no modifications. LD-Proofs represents
+a new approach which would require more work to achieve the same result.
+          </dd>
+          <dt id="pf28">
+PF28: No additional canonicalization required.
+          </dt>
+          <dd>
+Beyond base64 URL encoding JSON and JWT don't require any canonicalization to be transmitted
+on the wire. The JWS can be calculated on any data inside of the payload. This results in less
+computation, less complexity, and light-weight libraries compared to JSON-LD and LD-Proofs where
+canonicalization is required.
+          </dd>
+          <dt id="pf29">
+PF29: No Internet PKI required.
+          </dt>
+          <dd>
+JSON-LD and LD-Proofs rely on resolving external documents, e.g., <code>@context</code>. This means that
+a verifiable credential system would rely on existing Internet PKI to a certain extend and
+cannot be fully decentralized. A JWT-based system does not need to introduce this dependency.
+          </dd>
+          <dt id="pf30">
+PF30: No resolution of external documents needed.
+          </dt>
+          <dd>
+JSON-LD and LD-Proofs require the resolution of external documents, which leads to an
+increased network load for the verifier of a verifiable presentation. This needs to be
+mitigated through caching strategies.
```

Comment:
Verifiers can be designed (and have been already) such that hitting the network is never necessary. If you're referring to fetching public key material, e.g., via DID documents, this would hold true regardless of the proof format. I recommend striking this one and any other one related to retrieving external documents (PF22, PF29, PF30). I think the methods are the same on this point.

Reply:
@dlongley This is why the item mentions that additional steps are required to achieve this behaviour. Would you agree with this?